### PR TITLE
🚧Update contributor covenant badge

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](https://github.com/KinsonDigital/.github/blob/master/docs/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 <div align="center">
-    <img align="center" src="./Documentation/Images/casl-logo.png" height="96">
+    <a href="#"><img align="center" src="./Documentation/Images/casl-logo.png" height="96"></a>
```
Comment (KinsonDigitalAdmin) [inline README.md:2]: This needs to be a simple image tag, not a link.

```suggestion
    <img align="center" src="./Documentation/Images/casl-logo.png" height="96">
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
     <br />
 </div>
 
 <h1 style="border:0;font-weight:bold" align="center">CASL</h1>
 
 <div align="center">
 
-[![Prod Release PR Status Check](https://img.shields.io/github/actions/workflow/status/KinsonDigital/CASL/prod-release-pr-status-check.yml?color=2F8840&label=Prod%20CI%20Build&logo=GitHub)](https://github.com/KinsonDigital/CASL/actions/workflows/prod-release-pr-status-check.yml)
-[![Prev Release PR Status Check](https://img.shields.io/github/actions/workflow/status/KinsonDigital/CASL/prev-release-pr-status-check.yml?color=2F8840&label=Preview%20CI%20Build&logo=GitHub)](https://github.com/KinsonDigital/CASL/actions/workflows/prev-release-pr-status-check.yml)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/KinsonDigital/CASL/prod-release-pr-status-check.yml?color=2F8840&label=Prod%20CI%20Build&logo=GitHub)
```
Comment (KinsonDigitalAdmin) [inline README.md:10]: Convert back to a clickable markdown badge.  It was fine the way it was.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/KinsonDigital/CASL/prev-release-pr-status-check.yml?color=2F8840&label=Preview%20CI%20Build&logo=GitHub)
```
Comment (KinsonDigitalAdmin) [inline README.md:11]: Convert back to a clickable markdown badge.  It was fine the way it was.

File: README.md [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
 
-[![Codecov](https://img.shields.io/codecov/c/github/KinsonDigital/CASL?color=2F8840&label=Code%20Coverage&logo=codecov)](https://app.codecov.io/gh/KinsonDigital/CASL/tree/release%2Fv1.0.0)
+![Codecov](https://img.shields.io/codecov/c/github/KinsonDigital/CASL?color=2F8840&label=Code%20Coverage&logo=codecov)
```
Comment (KinsonDigitalAdmin) [inline README.md:13]: Convert back to a clickable markdown badge.  It was fine the way it was.

File: README.md [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
 
 [![Latest Nuget Release](https://img.shields.io/nuget/vpre/kinsondigital.CASL?label=Latest%20Release&logo=nuget)](https://www.nuget.org/packages/KinsonDigital.CASL)
-[![Nuget Downloads](https://img.shields.io/nuget/dt/KinsonDigital.CASL?color=0094FF&label=nuget%20downloads&logo=nuget)](https://www.nuget.org/stats/packages/KinsonDigital.CASL?groupby=Version)
+![Nuget](https://img.shields.io/nuget/dt/KinsonDigital.CASL?color=0094FF&label=nuget%20downloads&logo=nuget)
```
Comment (KinsonDigitalAdmin) [inline README.md:16]: Convert back to a clickable markdown badge.  It was fine the way it was.

File: README.md [hunk 6]
```diff
@@ -38,22 +38,23 @@ Currently, this library can play **.ogg** and **.mp3** file formats.  There are
...
   - Rewind
-  - Manual position set
+  - Manual set
 - Choosing audio hardware devices
 
 <h2 style="font-weight:bold;" align="center">🙏🏼 Contributing 🙏🏼</h2>
 
-Interested in contributing? If so, click [here](https://github.com/KinsonDigital/.github/blob/master/docs/CONTRIBUTING.md) to learn how to contribute your time or [here](https://github.com/sponsors/KinsonDigital) if you are interested in contributing your funds via a one-time or recurring donation.
+Interested in contributing? If so, click [here](https://github.com/KinsonDigital/.github/blob/master/docs/CONTRIBUTING.md) to learn how to contribute your time or [here](https://github.com/sponsors/KinsonDigital) if you are interested in contributing your funds via one-time or recurring donation.
+
```
Comment (KinsonDigitalAdmin) [inline README.md:57]: Remove this extra empty line

File: README.md [hunk 7]
```diff
@@ -65,11 +66,13 @@ Interested in contributing? If so, click [here](https://github.com/KinsonDigital
 
 <h2 style="font-weight:bold;" align="center">🚔 Licensing and Governance 🚔</h2>
 
+
```
Comment (KinsonDigitalAdmin) [inline README.md:69]: Remove this extra empty line

File: README.md [hunk 8]
```diff
@@ -65,11 +66,13 @@ Interested in contributing? If so, click [here](https://github.com/KinsonDigital
...
 <div align="center">
 
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg?style=flat)](https://github.com/KinsonDigital/.github/blob/master/docs/code_of_conduct.md)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg?style=flat)](https://github.com/KinsonDigital/.github/blob/master/docs/code_of_conduct.md)
 [![GitHub](https://img.shields.io/github/license/kinsondigital/CASL)](https://github.com/KinsonDigital/CASL/blob/release/v1.0.0/LICENSE.md)
 </div>
 
+
```
Comment (KinsonDigitalAdmin) [inline README.md:76]: Remove this extra empty line

File: README.md
Comment (KinsonDigitalAdmin) [inline README.md:1]: There is some extra space in the beginning of the 2 twitter badges.  Remove the spaces to bump them all the way to the left.

```diff
-  [![twitter-logo](https://raw.githubusercontent.com/KinsonDigital/.github/master/Images/twitter-logo-16x16.svg)Calvin Wilkinson](https://twitter.com/KDCoder) (KinsonDigital GitHub Organization - Owner)
+ [![twitter-logo](https://raw.githubusercontent.com/KinsonDigital/.github/master/Images/twitter-logo-16x16.svg)Calvin Wilkinson](https://twitter.com/KDCoder) (KinsonDigital GitHub Organization - Owner)
  
-  [![twitter-logo](https://raw.githubusercontent.com/KinsonDigital/.github/master/Images/twitter-logo-16x16.svg)Kristen Wilkinson](https://twitter.com/kswilky) (KinsonDigital GitHub Organization - Project Management, Documentation, Tester)
+ [![twitter-logo](https://raw.githubusercontent.com/KinsonDigital/.github/master/Images/twitter-logo-16x16.svg)Kristen Wilkinson](https://twitter.com/kswilky) (KinsonDigital GitHub Organization - Project Management, Documentation, Tester)
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
